res/values/config.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="restore_action" translatable="false">com.stevesoltys.seedvault.RESTORE_BACKUP</string>
```

Comment:
let's keep inside action class itself, for consistency with existing pattern (refer other action classes).
if needed (if at all) we can make a single such change for all actions. @quh4gko8

Reply:
is this resolved?

Reply:
Greying out the next button is a good idea

Reply:
> There's no need for config.xml for now, the activity that starts the backup utilizes intent action, not an explicit component. The showcase of dialog isn't preferred in comparison to hiding the activity, or greying out the next button

maybe along with greying out the button maybe there could be a textview added to the layout file and when the button is greyed out the textview will show, the text on it would be something along the lines of "restoring from a seedvault backup is currently unavailable"